pkg/lib/configreader/errors.go

```diff
@@ -121,6 +122,13 @@ func ErrorAlphaNumericDashDotUnderscore(provided string) error {
 	})
 }
 
+func ErrorValidDockerImage(provided string) error {
+	return errors.WithStack(&errors.Error{
+		Kind:    ErrValidDockerImage,
+		Message: fmt.Sprintf("%s is not a valid docker image resource", s.UserStr(provided)),
```

Comment:
nit: "%s is not a valid docker image path"

Reply:
Done!